requirements.txt

```diff
@@ -13,5 +13,6 @@ networkx
 pathlib
 nose
 nose-exclude
+coverage
 xlwt
 git+https://github.com/scikit-rf/scikit-rf.git@master
```

Comment:
this line has been removed in #459. Could you try updating your PR from the current master branch? (I don't understand why github is telling me there is no conflict)

Reply:
Sure. I just did that below. Should work now.